File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/sql/TestHigherMigration.java
package com.raizlabs.android.dbflow.test.sql;

import com.raizlabs.android.dbflow.sql.migration.Migration;
import com.raizlabs.android.dbflow.structure.database.DatabaseWrapper;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description:
 */
@com.raizlabs.android.dbflow.annotation.Migration(version = 1, database = TestDatabase.class, priority = 2)
public class TestHigherMigration implements Migration {
    @Override
    public void onPreMigrate() {

    }

    @Override
    public void migrate(DatabaseWrapper database) {

    }

    @Override
    public void onPostMigrate() {

    }
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/sql/TestHigherMigration.kt
package com.raizlabs.android.dbflow.sql

import com.raizlabs.android.dbflow.sql.migration.Migration
import com.raizlabs.android.dbflow.structure.database.DatabaseWrapper
import com.raizlabs.android.dbflow.TestDatabase

/**
 * Description:
 */
@com.raizlabs.android.dbflow.annotation.Migration(version = 1, database = TestDatabase::class, priority = 2)
class TestHigherMigration : Migration {
    override fun onPreMigrate() {

    }

    override fun migrate(database: DatabaseWrapper) {

    }

    override fun onPostMigrate() {

    }
}
